tests/test_load.py

```diff
@@ -0,0 +1,28 @@
+from google.cloud.storage import Client
+from google.cloud.storage.bucket import Bucket
+from google.cloud.storage.blob import Blob
+
+from crypto.load import Load
+
+
+def test_bucket(mock_gcs_client, mock_bucket):
+    load = Load(client=mock_gcs_client, bucket_name="test-bucket")
```

Comment:
was this easier to test with the dependency injection?

Reply:
btw should we make unit vs functional subdirectories to indicate that these tests use a lot of mocking (!= end-to-end)?

Reply:
yes, it was much easier to test using dependency injection. Wasn't using it and took a couple days to make some progress but ultimately dead end. Changed last night and a lot easier.

Reply:
yes, we should. I think I removed the unit subdirectory I had in place in yesterday's PR. I'll be sure to add them.

Reply:
ok cool, yep I like that approach and yes, dependency injection for the win (used it in the clinic yesterday as well to inject a test db into the functional test)